### PR TITLE
creat periodic table

### PR DESCRIPTION
**Pull request by catakul**
Hey, 

thanks for reviewing my code! Please note that the file "PeriodicTableJSON.json" which has over 4000 lines of code, does not need a to be reviewed.
The data is copied and only used to map data to create und fill in information needed for my periodic table of elements.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script src="index.js" />
+    <script src="index.js"></script>
```
Comment (Sven-Lyco) [inline public/index.html:10]: Why did you add this line? 🤔 
```suggestion

```

File: src/App.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
-export default function App() {
-  return <div>App goes here</div>;
+import React from "react";
```
Comment (Sven-Lyco) [inline src/App.js]:
```suggestion

```

File: src/App.js [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+import "./App.css";
+import PeriodicTable from "./Components/PeriodicTable.js";
+
+function App() {
```
Comment (Sven-Lyco) [inline src/App.js]: You can do it like this, if you want
```suggestion
export default function App() {
```

File: src/App.js [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+  return (
+    <div className="App">
+<h1>Periodic Table of Elements</h1>
+<PeriodicTable />
+    </div>
+  );
 }
+
+export default App;
```
Comment (Sven-Lyco) [inline src/App.js]:
```suggestion

```

File: src/Components/PeriodicTable.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+
+  }
+  
+  .element:hover {
+    transform: scale(1.25, 1.25);
+    z-index: 1;
```
Comment (Sven-Lyco) [inline src/Components/PeriodicTable.css:23]: Do you really need this on hover? 🤔 
```suggestion

```

File: src/Components/PeriodicTable.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
```
Comment (Sven-Lyco) [inline src/Components/PeriodicTable.js]:
```suggestion

```

File: src/Components/PeriodicTable.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    "alkaline earth metal": "#EC674E",
+    "diatomic nonmetal": "#D81159",
+    "alkali metal": "#8F2D56",
+    "transition metal": "#58586B",
+    "post-transition metal": "#218380",
+    "polyatomic nonmetal": "#40FF4D",
+    lanthanide: "#4AABAF",
+    metalloid: "#73D2DE",
+    
+  };
```
Comment (Sven-Lyco) [inline src/Components/PeriodicTable.js]: I think you shouldn't use whitespaces as a key 🤔 
```suggestion
  const colorMap = {
    "actinide": "#7377FA",
    "noble-gas": "#FFBC42",
    "alkaline-earth-metal": "#EC674E",
    "diatomic-nonmetal": "#D81159",
    "alkali-metal": "#8F2D56",
    "transition-metal": "#58586B",
    "post-transition-metal": "#218380",
    "polyatomic-nonmetal": "#40FF4D",
    "lanthanide": "#4AABAF",
    "metalloid": "#73D2DE",
  };
```

**Review by Sven-Lyco — APPROVED** [review]
Good job, I made you a few suggestions 🚀

File: public/index.html (hunk 1 above)
Comment (catakul) [reply public/index.html:10]: Because our coaches told us to do so. 
Later it turned out to be irrelevant since react is handling the script command anyway.

File: src/Components/PeriodicTable.css (hunk 5 above)
Comment (catakul) [reply src/Components/PeriodicTable.css:23]: Yes the hover highlights (at least on desktop) each Element, it just looks clean. And since I am doing the hover effect, the z-index is necessary. Otherwise the boxes of each element are overlapping while the hover effect is active.

File: src/Components/PeriodicTable.js (hunk 7 above)
Comment (catakul) [reply src/Components/PeriodicTable.js]: makes sense. Thank you! ✌🏼
